### ISSUE TITLE
Full playbook and roles

### ISSUE THREAD

**Issue by arsenicks** [mkell43/ansible-docker-xmrig-miner#1]
Hey, thanks for sharing. I took a few other roles shared on github to complete the setup and make it idempotent. Would you be interested to see and update ?

I'm still testing it but it should work great on Debian and Redhat based distro.

**Comment by mkell43** [comment]
Looks pretty good, I'm not an expert, but I have a few suggestions for it.  Mind if I shoot you a message to the email I saw in your commits?  Otherwise you can find me on Telegram under the same username I use here or on Reddit with the same username as well.

**Comment by arsenicks** [comment]
Sure send me an email! Same here, no expert, I'm just starting with ansible, but I managed to get everything working on both Centos and Ubuntu.

Anybody having a little bit of knowledge on ansible will probably loose their ability to see but hey, I'm learning ;) I tried to fork yesterday but wasn't sure of what I was doing.. I'm not that used to play with git.

So I'll read a little bit more tonight and if I'm right I should be able to fork, push my modif and ask for a pull request. Anyway send me a mail with your suggestions! ;)
